champions/src/main/java/me/mykindos/betterpvp/champions/Champions.java

```diff
@@ -75,13 +77,15 @@ public void onEnable() {
             var itemHandler = injector.getInstance(ItemHandler.class);
             itemHandler.loadItemData("champions");
 
-            var weaponManager = injector.getInstance(WeaponManager.class);
-            weaponManager.load();
-
             var recipeHandler = injector.getInstance(RecipeHandler.class);
             recipeHandler.loadConfig(this.getConfig(), "champions");
 
             updateEventExecutor.loadPlugin(this);
+
+            final Adapters adapters = new Adapters(this);
+            final Reflections reflectionAdapters = new Reflections(PACKAGE);
+            adapters.loadAdapters(reflectionAdapters.getTypesAnnotatedWith(PluginAdapter.class));
+            adapters.loadAdapters(reflectionAdapters.getTypesAnnotatedWith(PluginAdapters.class));
```

Comment:
Is this still necessary

Reply:
yes, it loads plugin adapters without explicitly calling the class which loads its definition and could cause exceptions if we reference it directly